.github/workflows/ci.yaml

```diff
@@ -0,0 +1,58 @@
+name: ci
+on:
+  - push
+  - pull_request
+env:
+  golang-version: '1.13'
+  kind-version: 'v0.9.0'
+jobs:
+  generate:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os:
+          - macos-latest
+          - ubuntu-latest
+    name: Generate
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-go@v2
+      with:
+        go-version: ${{ env.golang-version }}
+    - run: make --always-make generate && git diff --exit-code
+  unit-tests:
+    runs-on: ubuntu-latest
+    name: Unit tests
+    steps:
+    - uses: actions/checkout@v2
+    - run: make --always-make test
+  e2e-tests:
+    name: E2E tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        kind-image:
```

Comment:
Can we somehow have this in `env` section? This would allow more visibility.

Reply:
I could not get this to work, the env section seems to only allow string key-value pairs. Do you of a hack that could make this possible?

Reply:
Nope, I don't have anything. Let's leave it as is if GH actions do not support it.